Makefile

```diff
@@ -2,6 +2,7 @@ all: dependencies templates
 
 # The release version of `dd-sdk-swift-testing` to use for tests instrumentation.
 DD_SDK_SWIFT_TESTING_VERSION = 2.3.2
+DD_DISABLE_TEST_INSTRUMENTING = true
```

Comment:
had to disable for the moment 

backlog RUM-3185

Reply:
probably worth having a code comment with ticket reference as well

Reply:
suggestion/ The `RUM-3185` gives enough context on why we disable it 👍. Let's include that JIRA in the PR description, so it is clear why we had to turn it off.